### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 39

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#215] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255244-70cf33f0-5e1d-4976-8769-766f339a5467.png)

**Comment by i0nut02** [comment]
1. (25 scelgo 10)
2. (15 scelgo 4) * (10 scelgo 6) -------> non sono sicuro sia giusta
3. (25 scelgo 10) - (15 scelgo 10) - (10 scelgo 10)

**Comment by ATLANCTIX4** [comment]
@i0nut02 @matypist

Il terzo punto potrebbe essere risolto in questo modo?:

(10 scelgo 1) * (15 scelgo 1) * (23 scelgo 8)

**Comment by i0nut02** [comment]
di base hanno un risultato diverso, penso che questo sia dovuto principalmente al fatto che poni una sorta di ordine, infatti il tuo ha un numero di combinazioni maggiore
